### PR TITLE
Comment all the things

### PR DESCRIPTION
**Pull request by ooterness**
Fix missing or inconsistent documentation for many clock faces:
* Move from .c to .h as needed for consistency.
* When missing from both, copy from pull request or wiki.
* When missing entirely, infer functionality from source code.
* In a few cases, add missing license boilerplate.

This PR should be 100% comment changes only, no functional impact whatsoever.

### CODE REVIEW

**Comment by ooterness** [comment]
Sometime soon, I'll go and resync new comments here with what's available on the wiki.

**Comment by ooterness** [comment]
@josecastillo : This has been open for a few days now. Any comments or concerns?

File: movement/watch_faces/clock/beats_face.c [hunk 1]
```diff
@@ -1,3 +1,27 @@
+/*
+ * MIT License
```
Comment (WesleyAC) [inline movement/watch_faces/clock/beats_face.c:2]: Adding copyright info to code you didn't write seems dubious to me. @mrbrown8, can you confirm this?

File: movement/watch_faces/clock/wyoscan_face.h [hunk 2]
```diff
@@ -1,7 +1,7 @@
 /*
  * MIT License
  *
- * Copyright (c) 2023 <#author_name#>
+ * Copyright (c) 2023 Willy Hardy <https://github.com/wjhrdy>
```
Comment (WesleyAC) [inline movement/watch_faces/clock/wyoscan_face.h:4]: @wjhrdy can you confirm this?

File: movement/watch_faces/complication/discgolf_face.c [hunk 3]
```diff
@@ -1,7 +1,31 @@
+/*
+ * MIT License
+ *
+ * Copyright (c) 2023 Þorsteinn Jón Gautason
```
Comment (WesleyAC) [inline movement/watch_faces/complication/discgolf_face.c:4]: @RighteousLightning could you confirm this?

File: movement/watch_faces/complication/totp_face.c [hunk 4]
```diff
@@ -1,19 +1,34 @@
+/*
+ * MIT License
+ *
+ * Copyright (c) 2022 Wesley Ellis (https://github.com/tahnok)
```
Comment (WesleyAC) [inline movement/watch_faces/complication/totp_face.c:4]: @tahnok can you confirm this?

File: movement/watch_faces/complication/totp_face_lfs.c [hunk 5]
```diff
@@ -1,3 +1,27 @@
+/*
+ * MIT License
+ *
+ * Copyright (c) 2022 James Haggerty (https://github.com/wryun)
```
Comment (WesleyAC) [inline movement/watch_faces/complication/totp_face_lfs.c:4]: @wryun can you confirm this copyright?

File: movement/watch_faces/settings/preferences_face.h [hunk 6]
```diff
@@ -25,6 +25,56 @@
...
+ *
+ *  LT - Light.
+ *      This setting has three screens.
+ *      The first lets you choose how long the LED should stay lit when the
+ *       LIGHT button is pressed. Options are 1 second, 3 seconds and 5
+ *       seconds, or “No LED” to disable the LED entirely.
+ *      The second screen, titled “blu”, sets the intensity of the blue LED.
+ *       Values range from 0 (off) to 15 (full intensity)
+ *      The third screen, “red”, sets the intensity of the red LED, again
+ *       from 0 to 15.
```
Comment (WesleyAC) [inline movement/watch_faces/settings/preferences_face.h:71]: This will be different depending on the board, might be worth mentioning that.

**Review by WesleyAC — CHANGES_REQUESTED** [review]
Thanks so much for this! It's really great to see everything in one place.

I'm dubious about putting in the copyright names without confirmation from those people, but if you remove that I can go ahead and merge this, and we can add that back as people confirm.

File: movement/watch_faces/clock/beats_face.c (hunk 1 above)
Comment (mrbrown8) [reply movement/watch_faces/clock/beats_face.c:2]: Thank you for asking, but I did NOT write this code. I cannot take credit/blame for this.

**Comment by tahnok** [comment]
Beats, totp face are MIT licensed

File: movement/watch_faces/clock/beats_face.c (hunk 1 above)
Comment (ooterness) [reply movement/watch_faces/clock/beats_face.c:2]: @WesleyAC you are right.  In hindsight, that was awfully presumptuous of me.  Sorry!

For the Beats face, @tahnok is the primary author and has confirmed MIT licensing info in a comment below.  I don't know what I was thinking when I put @mrbrown8 , but I will fix that now.

**Comment by ooterness** [comment]
@tahnok has confirmed MIT license is correct for Beats and TOTP.  That leaves Discgolf, TOTP-LFS, and Wyoscan.

I'll leave this as-is for a few days in case those authors respond.  If not, I'll revert the changes to those faces so we can move forward.  I can open a separate issue to get copyright questions straightened out.

File: movement/watch_faces/complication/totp_face_lfs.c (hunk 5 above)
Comment (wryun) [reply movement/watch_faces/complication/totp_face_lfs.c:4]: This is heavily based on @tahnok 's code, so I'd just use the notice from totp_face.c

Comment (ooterness) [reply movement/watch_faces/complication/totp_face_lfs.c:4]: Acknowledged, I'm making the change now.

**Comment by ooterness** [comment]
I've reverted the remaining unconfirmed copyright-related changes.  (i.e., Discgolf and Wyoscan.)

**Pull request merged**